src/renderer/domains/network/identity/store.ts

```diff
@@ -64,8 +64,12 @@ const requestFx = attach({
 
 const requestWithRetryFx = createEffect<Omit<FetchParams, 'api'>, Record<AccountId, AccountIdentity>>(
   ({ chainId, accounts }) => {
-    const binded = scopeBind(requestFx, { safe: true });
-    return fetchPool.call(() => binded({ chainId, accounts }));
+    try {
+      const binded = scopeBind(requestFx, { safe: true });
```

Comment:
`bind` is irregular verb > `bind | bound | bound`
```suggestion
      const boundRequest = scopeBind(requestFx, { safe: true });
```